src/components/MapSelectors/GlobalSelector.js

```diff
@@ -235,10 +235,24 @@ const GlobalSelector = ({ handleChange, extendRight }) => {
     }
   };
 
+  const escapeKeyStateReducer = (state, changes) => {
+    console.log(changes);
```

Comment:
Perhaps remove this?  Or else add some surrounding text so somebody seeing it in the console knows what it is.

Reply:
I meant to remove this, thanks!